server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/SendMDNProcessor.java

```diff
@@ -151,7 +151,7 @@ private MessageId sendMdn(ValueWithId.MDNCreationEntry MDNCreationEntry, Mailbox
         Message mdnAnswer = mdn.generateMDNMessage(originalMessage, mailboxSession);
 
         Flags seen = new Flags(Flags.Flag.SEEN);
-        MessageFactory.MetaDataWithContent metaDataWithContent = messageAppender.appendMessageInMailbox(mdnAnswer,
+        MessageFullViewFactory.MetaDataWithContent metaDataWithContent = messageAppender.appendMessageInMailbox(mdnAnswer,
```

Comment:
Can we import MessageFullViewFactory.MetaDataWithContent so that we don't have the MessageFullViewFactory prefix?